assets/Illustrator Files/RefineryGraphics261119/RefineryGraphics261119 Report.txt

```diff
@@ -0,0 +1,168 @@
+Package Report
```

Comment:
this file is not useful and contains references to your individual computer and `Dropbox` folder. Please remove this from PR.